docs/development-and-publish-tasks.md

```diff
@@ -59,6 +59,18 @@ This task will:
 This task will:
 - will run accessibility test (tenon and axe)
 
+**`gulp test:app`**
+
+This task will:
```

Comment:
Duplication of 'will' – which I notice is also a problem above! Would be good to fix this throughout.